frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -21,12 +28,26 @@ import {
   dataVolumeManifest,
 } from './utils/mocks';
 
+function resolveStorageDataAttribute(configMap: any, attribute: string): string {
+  if (Object.prototype.hasOwnProperty.call(configMap, `${STORAGE_CLASS}.${attribute}`)) {
```

Comment:
we could also use lodash `_.has` here and make a const out of `${STORAGE_CLASS}.${attribute}`

Reply:
Done